hardware_interface/CMakeLists.txt

```
@@ -1,54 +1,76 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(hardware_interface)
 
-find_package(catkin REQUIRED COMPONENTS roscpp)
+# Load catkin and all dependencies required for this package
+find_package(catkin REQUIRED COMPONENTS
+  roscpp
+)
 
-# Declare catkin package
+# Include header-only Boost libraries
+find_package(Boost REQUIRED)
```

off of the top of my head, this could read something similar
```suggestion
find_package(Boost REQUIRED COMPONENTS PointerContainer)
```

---

According to the first comment on [this SO answer](https://stackoverflow.com/a/6646518), you don't need to `find_package` any components which are header-only (Such as pointer container and lexical cast)

---

That makes a lot of sense, I'll keep this in mind. Thanks!